sql/testdata/txn

```diff
@@ -0,0 +1,75 @@
+statement ok
+CREATE TABLE kv (
+  k CHAR PRIMARY KEY,
+  v CHAR
+)
+
+statement ok
+INSERT INTO kv (k,v) VALUES ('a', 'b')
+
+query TT
+SELECT * FROM kv
+----
+a    b
```

Comment:
FYI, you only need whitespace for visual formatting. I usually only use 1 space in simple results like this.